main.php

```diff
@@ -71,6 +70,21 @@
 		exit(1);
 	}
 }
+// Allow a first class env var to be set that disables .env file loading
+if (!getenv('SS_IGNORE_DOT_ENV')) {
```

Comment:
Hmm - I had moved this for a reason - but I think that was to do with the host issue that I've now fixed. I'll move it back to Constants.php and see if I remember why I moved it :)